package.json

```diff
@@ -1,21 +1,17 @@
 {
-  "name": "azure-rest-api-specs-tests",
-  "private": true,
-  "author": {
-    "name": "Microsoft Corporation",
-    "email": "azsdkteam@microsoft.com",
-    "url": "https://github.com/azure/azure-rest-api-specs"
-  },
-  "version": "0.1.0",
-  "description": "Tests for Azure REST API Specifications",
-  "license": "MIT",
   "dependencies": {
     "@azure-tools/cadl-autorest": "0.25.0",
     "@azure-tools/cadl-azure-core": "0.25.0",
     "@azure-tools/cadl-azure-resource-manager": "0.25.0",
     "@azure-tools/cadl-providerhub": "0.25.0",
     "@azure-tools/cadl-apiview": "0.3.5",
-    "@cadl-lang/compiler": "0.39.0"
+    "@cadl-lang/compiler": "0.39.0",
```

Comment:
You mentioned in the meeting having Cadl/TypeSpec side-by-side for now. I had thought that Cadl compiler was on a much earlier version. As it stands now with `cadl` at 0.39, I don't think there is a need for it since `@typespec/compiler 0.42.0` fully back compat with `@cadl\compiler 0.39`. It should compile any Cadl folder not yet migrated as it is.